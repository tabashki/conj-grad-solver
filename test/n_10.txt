-4.3833004387e+00
-7.8121859428e+00
1.7168261935e+01
-9.1101167770e+00
-5.7046450721e+00
1.0332770017e+01
-4.7808780574e+00
8.3259061655e-01
-8.8287110083e+00
1.2900498716e+01
