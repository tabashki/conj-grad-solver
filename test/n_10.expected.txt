4.0467348960e+00
3.7101693534e+00
-4.4385821321e+00
4.5809283177e+00
4.4903219904e+00
-1.3049294090e+00
3.2325892089e+00
2.9892297694e+00
3.5784609464e+00
-4.6610188848e+00
